notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<p>This is a case where we are interested in seeing Elevation only. As the log message says, we are going from elevation 20º up to 80º and pressing the e-stop halfway. Azimuth is not used here and we can remove for this case. </p><p>I suggest we do the same for the other cases. They are usually interesting in one of the two axes. </p><p> </p><p>I suggest you try to isolate the test too. Here, for example, we are interested in the slew that goes from 20º to 80º. Everything before and after that is just noise for the reader. Maybe start a few seconds before the slew begin and go a few seconds after it is complete. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>